addons/binding/org.openhab.binding.freebox/README.md

```diff
@@ -0,0 +1,63 @@
+# Freebox Binding Configuration
+
+This binding integrates the [Freebox Revolution](http://www.free.fr/adsl/freebox-revolution.html) to your openHab installation.
+
+![Freebox Revolution](doc/freebox-revolution.jpg)

```

Comment:
Do you own the rights on this picture? Otherwise, better remove it or replace it by one that you own.